### ISSUE TITLE
assets/dicts not committed to repo

### ISSUE THREAD

**Issue by shirakaba** [farshed/duofolio#5]
I imagine these files haven’t been committed due to licensing issues or file size; is that the case?
Also is there a TypeScript interface describing its contents somewhere?

**Comment by rafaelsaback** [comment]
Indeed, I tried to run the project locally, but it's failing due to the missing dictionaries. I'll be glad to make contributions once I'm able to run it locally :slightly_smiling_face: 

Regarding the TypeScript question, the project doesn't seem to have anything TS-related, it's purely JavaScript. I confess I'm a big TS fan and I feel that as the project scales, it would be nice to add types to it (actually, the sooner the better).

**Comment by farshed** [comment]
Hey @shirakaba. Yes, licensing terms on dictionary files don't allow me to share them along the source code. I'm currently rewriting the translator feature from scratch so it can support more languages and wouldn't need to be dependant on third-party dictionary data.

@rafaelsaback I don't share your sentiments about TypeScript 😅 but I'd be willing to merge any TS-related pull requests :) I just think it would be better if you hold off it for the moment till the next update is released. lt's a very big one and I'm hoping to push it by the end of the week.

**Comment by rafaelsaback** [comment]
Hahahaha

Thanks for the heads up @farshed, I'll be looking forward to the new release 🙂

**Comment by coreyar** [comment]
Not related to the original issue but I want to offer my support via contributions. I enjoy using Typescript and find it helpful in creating robust applications. Happy to contribute to adding Typescript or another issue :)
